roles/kubernetes/preinstall/tasks/resolvconf.yml

```diff
@@ -4,17 +4,33 @@
   register: resolvconf
   ignore_errors: yes
   changed_when: false
+  tags: facts
+
+- name: check kubelet
```

Comment:
@mattymo this doesn't look like the best check ever, but the point is to figure out if the node has yet K8s installed or yet the cluster member. Then, the DNS resolv (/etc/resolv.conf) must be configured w/o k8s dns_server record.